Identification-and-Assessment.tex

```diff
@@ -30,49 +30,88 @@ \subsection{Create Risk Item}
 	\item Parent --- Any applicable Parent risks
 \end{itemize}
 
+Risks can be created as ``children'' of ``parent'' risks.
+This is appropriate for a risk that is anticipated as being realized in separate time-ordered phases, or as distinct parallel events that each have their own impact.
+The cost and schedule impact of the parent is the combined cost and schedule impact of its children: see the next section for details.
+In order for the combined impact to be simple to understand and estimate, child risks must be constructed to be fully independent (non-overlapping) and a complete set.
+(Risk owners who are considering breaking down a risk into component children should only make that move if they can satisfy these criteria.)
+
 \subsection{RROB Candidate Risk Review}
 
-Should be reviewed by POC before submitting.
-RROB to review for sufficiency and accuracy of information for managing group for ability to manage effectively as an Active risk.
+Should be reviewed by the Risk Owner before submitting.
+RROB to review for sufficiency and accuracy of information, to confirm ability of Risk Owner to manage effectively as an Active risk.
 
-Minimum content before submitting to RROB:
+Minimum content before submitting a Candidate Risk for review by the RROB:
 \begin{itemize}
-	\item Cost Impact --- According to risk table (need to add risk table to RDO-71)
+	\item Cost Impact --- According to risk table (need to add risk table to RDO-71).
 	\item Likelihood --- According to risk table (need to add risk table to RDO-71)
 	\item Schedule Impact --- According to risk table (need to add risk table to RDO-71)
-	\item Cost or Schedule Impact Description, if over certain severity.
-	\item Cost or Schedule fields under Analyze Risk Quantitative, if over certain severity.
-	\item Plan Type, and a minimum of one related Reponse Plan
-	\item Realized Risk Plan --- Description of event trigger
+	\item Cost and/or Schedule Impact Description
+	\item Cost and/or Schedule fields under Analyze Risk Quantitative, if impact is non-negligible.
+	The Likely Cost is the estimated cost of addressing the risk in the year after it is realized.
```

Comment:
The Likely Cost is also a Residual (ie post-mitigation) quantity, even though its name does not include the word "Residual". 
```suggestion
	The Likely Cost is the estimated post-mitigation cost of addressing the risk in the year after it is realized.
```